wsd/RequestVettingStation.cpp

```diff
@@ -304,6 +304,10 @@ void RequestVettingStation::handleRequest(const std::string& id,
                           << docKey << ']');
 
             _ws->sendMessage("browsersetting: null");
```

Comment:
I imagine the intent was to put this inside the isUnitTesting and not duplicate it

Reply:
Checking ..

Reply:
The first is a raw WebSocket connection established before session creation, while the second uses the session instance, making them distinct. We can consider improvements after the unit tests pass.